packages/web/src/pages/upload-page/fields/download-availability/DownloadAvailability.tsx

```diff
@@ -92,24 +109,40 @@ export const DownloadAvailability = ({
     submitForm()
   }, [setStatus, submitForm])
 
-  const options = [
+  const isFollowersOptionDisabled =
+    !isUpload && !isContentFollowGated(initialDownloadConditions)
+  const isPremiumOptionDisabled =
+    !isUpload && !isContentUSDCPurchaseGated(initialDownloadConditions)
+  const options: Option<DownloadTrackAvailabilityType>[] = [
     {
       key: DownloadTrackAvailabilityType.PUBLIC,
       text: messages.public,
-      icon: <IconVisibilityPublic size='s' fill={color.neutral.neutral} />
+      icon: <IconVisibilityPublic size='s' fill={neutral} />
     },
     {
       key: DownloadTrackAvailabilityType.FOLLOWERS,
       text: messages.followers,
-      icon: <IconUserFollowing size='s' fill={color.neutral.neutral} />,
-      disabled: !isUpload && !isContentFollowGated(initialDownloadConditions)
+      icon: (
+        <IconUserFollowing
+          size='s'
+          fill={isFollowersOptionDisabled ? subdued : neutral}
+        />
+      ),
+      disabled: isFollowersOptionDisabled
     },
     {
       key: DownloadTrackAvailabilityType.USDC_PURCHASE,
       text: messages.premium,
-      icon: <IconCart size='s' fill={color.neutral.neutral} />,
-      disabled:
-        !isUpload && !isContentUSDCPurchaseGated(initialDownloadConditions)
+      icon: (
+        <IconCart
+          size='s'
+          fill={
+            isPremiumOptionDisabled || !isUsdcUploadEnabled ? subdued : neutral
+          }
+        />
+      ),
+      disabled: isPremiumOptionDisabled,
```

Comment:
does this also need to incorporate the ff? `disabled: isPremiumOptionDisabled || !isUsdcUploadEnabled`?

Reply:
no. we want it to be enabled/clickable, and show the waitlist callout when it's selected